### PR TITLE
Play list

### PR DESCRIPTION
**Pull request by ragini07**
1.Create Playlist / Delete Playlist 
2.Add video to playlist / delete video from playlist

### CODE REVIEW

File: src/Components/PlayLists/PlayList.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    background-color: var(--secondary-color);
+    max-width: 10rem;
+    max-height: 10rem;
+    color: var(--lg-color);
+    padding: 0.4rem 0.4rem;
+    cursor: pointer;
+}
+.playlist-text{
+  
+}
```
Comment (ScoobyDubeyDo) [inline src/Components/PlayLists/PlayList.css]: Extra  code , can be deleted

File: src/Components/PlayLists/PlayListModal.js [hunk 2]
```diff
@@ -1,9 +1,91 @@
-import React from 'react'
+import {useState} from 'react'
+import {useAuth} from '../../Context/auth-context'
+import {useUser} from '../../Context/user-context'
+import {useVideos} from '../../Context/data-context'
```
Comment (znjs) [inline src/Components/PlayLists/PlayListModal.js]: can use barrel export

File: src/Components/PlayLists/SinglePlayList.js [hunk 3]
```diff
@@ -1,9 +1,65 @@
...
+    //             setIsLoading(true)
+    //            const {data , status}  = await axios.get(`/api/user/playlists/${id}`)
+    //            setPlayListVideos(data.videos)
+    //            setIsLoading(false)
+    //         }catch(error){
+    //             console.log("error in getting video from playlist", error)
+    //             setIsLoading(false)
+    //         }
+    //     })()
+    // },[])
```
Comment (znjs) [inline src/Components/PlayLists/SinglePlayList.js]: can remove the commented code

File: src/Components/PlayLists/PlayListModal.js (hunk 2 above)
Comment (ScoobyDubeyDo) [inline src/Components/PlayLists/PlayListModal.js]: **suggestion**: use barrel export

**Review by znjs — APPROVED** [review]
- Good folder Strutures
- Good naming conventions used
- Liked the modular approach of the file system
- Preview working as expected

File: src/Components/PlayLists/SinglePlayList.js (hunk 3 above)
Comment (ScoobyDubeyDo) [inline src/Components/PlayLists/SinglePlayList.js]: **Must**:  Delete the unused code

**Review by ScoobyDubeyDo — APPROVED** [review]
Functioning well, just gave a few suggestions look into it

**Review by aniketxparihar — APPROVED** [review]
Liked the Folder Structure
Liked the CSS naming
Indentations can be better try to use prettier
rest everything looks good

**Pull request merged**